app/shared/translate/lang-de.ts

```diff
@@ -0,0 +1,10 @@
+﻿/**
+ * German Tanslations goes here.
+ */
+
+export const LANG_DE_NAME = 'de';
+
+export const LANG_DE_TRANS = {
+    'hello world': 'hallo welt',
+    'login title': 'Geben Sie Ihre Daten einloggen',
```

Comment:
okay, I just relied on Google for this :P . I just gave an example on how to use this. It is subjected to change.